index.html

```diff
@@ -212,27 +192,27 @@ <h2>Extension to MediaStreamTrack</h2>
     <section>
       <h3>Audio Content Hints</h3>
       <p>
-        Audio content hints are only applicable when the <a>MediaStreamTrack</a>
+        Audio content hints are only applicable when the {{MediaStreamTrack}}
         contains an audio track.
       </p>
       <table class="simple"><tbody><tr><th colspan="2">Audio content hints</th></tr>
         <tr><td><code>""</code></td><td><p>
           No hint has been provided, the implementation should make its
           best-informed guess on how to handle contained audio data. This may be
           inferred from how the track was opened or by doing content analysis.
-        <tr><td><code id="idl-def-AudioContentHint.speech">speech</code></td><td><p>
+        <tr><td><dfn id="idl-def-AudioContentHint.speech"><code>"speech"</code></dfn></td><td><p>
           The track should be treated as if it contains speech data. Consuming
           this signal it may be appropriate to apply noise suppression or boost
           intelligibility of the incoming signal.
         </p></td></tr>
-        <tr><td><code id="idl-def-AudioContentHint.speechRecognition">speechRecognition</code></td><td><p>
+        <tr><td><dfn id="idl-def-AudioContentHint.speechRecognition"><code>"speechRecognition"</code></dfn></td><td><p>
```

Comment:
This one is changing to "speech-recognition" per other PR.

Reply:
right - I'm happy to integrate that PR in this one, or we can manage the conflict once one of them gets merged